akka-http-core/src/test/scala/akka/http/impl/engine/client/TlsEndpointVerificationSpec.scala

```diff
@@ -47,6 +51,12 @@ class TlsEndpointVerificationSpec extends AkkaSpec("""
         e shouldBe an[Exception]
```

Comment:
Ah, it seems to be a wrapper around `sun.security.util.HostnameChecker` so the JDK 6 already contains a Hostname checker but it doesn't wire it into the SSLEngine. Interesting.